src/adapters/postgres/user-adapter.ts

```
@@ -760,13 +761,67 @@ export class PostgresUserService implements IServicelocator {
 
-  async updateUser(userDto, response: Response) {
+  async updateUser(request: any, userUpdateDto:UserUpdateDTO, response: Response) {
     const apiId = APIID.USER_UPDATE;
     try {
       const updatedData = {};
       const editIssues = {};
+      let errKeycloak = "";
+      let resKeycloak;
+
+      //If usernaem is exist in update DTO
+      if(userUpdateDto.userData && userUpdateDto.userData.username){
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)